packages/css/src/grid/grid.scss

```diff
@@ -28,6 +28,14 @@
   --amsterdam-grid-padding-inline: var(--amsterdam-grid-compact-padding-inline);
 }
 
+.amsterdam-grid--gap-vertical--small {
+  row-gap: calc(var(--amsterdam-grid-gap) / 2);
```

Comment:
So you'll only allow a natural flow of grid cells into rows and not `grid-template-columns`? Or are developers free to add their own CSS to force cells into rows like I'm doing to the Header? Fidgeting with just empty cells, start and span is not my idea of a good time.